src/blob/handlers/BlobHandler.ts

```diff
@@ -344,7 +388,6 @@ export default class BlobHandler extends BaseHandler implements IBlobHandler {
       options.modifiedAccessConditions
     );
 
-    // ToDo: return correct headers and test for these.
```

Comment:
Please add back the TODO comment, until you have finished it.

Reply:
This comment is from before my changes. Just reverted the deletion